changelog.d/14880.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug when using the `send_local_online_presence_to` module API.
```

Comment:
I'm not really sure of the impact of this bug -- sorry for the weak changelog. Any suggestions appreciated!